opm/input/eclipse/EclipseState/Grid/FieldPropsManager.hpp

```diff
@@ -239,6 +240,11 @@ class FieldPropsManager {
 
     const std::unordered_map<std::string,Fieldprops::TranCalculator>& getTran() const;
 
+    static constexpr std::string_view getMultiplierPrefix()
+    {
+        using namespace std::literals;
+        return "__MULT__"sv;
+    }
```

Comment:
Is there a reason that this function is public in the `FieldPropsManager` instead of a private implementation detail in `FieldProps.cpp`?

Reply:
HIstorically yes (needed in EclipseState when processing SCHEDULE). I'll change that back, then....